08-communicating-results.Rmd

```diff
@@ -44,66 +49,64 @@ anes_des <- anes_adjwgt %>%
- 3. **[CANDITATE NAME] for president**: This provides the information on the estimate. The number is the percentage of those voting for a specific candidate for a specific office.
- 4. **[YEAR] general election**: As with the bullet above, this information provides more context around the specific election and year. The estimate would take on a different meaning if we changed it to a *primary* election instead of a *general* election, for example.
+ 1. **[DATE]**: Given that polling data is time-specific, providing the date of reference lets the audience know when this data was valid.
+ 2. **Registered U.S. voters**: This tells the audience who we surveyed, letting them know the target population.
+ 3. **XX%**: This part provides the estimated percentage of people voting for a specific candidate for a specific office.
+ 4. **[YEAR] general election**: As with the bullet above, adding this gives more context about the election type and year. The estimate would take on a different meaning if we changed it to a *primary* election instead of a *general* election.
  
-This sentence also includes the word "estimated." When presenting results in aggregate from surveys, it is important not to talk about estimates in the absolute as we have errors around each estimate. Using words like "estimated," "on average," or "around" will help convey the uncertainty with a given value. Including that uncertainty would be more informative to the audience. For example, a sentence could include uncertainty with a margin of error such as "XX% (+/- Y%)". Confidence intervals can also be incorporated into the text to assist readers.  
+We also included the word "estimated." When presenting aggregate survey results, we have errors around each estimate. We want to convey this uncertainty rather than talk in absolutes. Words like "estimated," "on average," or "around" can help our communication to the audience. Instead of saying 'XX%,' we can say 'XX% (+/- Y%)' to show the margin of error. Confidence intervals can also be incorporated into the text to assist readers.  
 
-Second, providing context and discussion around the *meaning* of the point can help the audience glean some insight into why the data is important. For example, when comparing two points, it could be helpful to indicate that there are statistically significant differences and the impact and usefulness of this information. This is where it is crucial as researchers to do our best to keep biases in check and present only the facts logically. 
+Second, providing context and discussing the *meaning* behind a point can help the audience glean some insight into why the data is important. For example, when comparing two points, it can be helpful to highlight if there are statistically significant differences and explain the impact and relevance of this information. This is where we, as analysts, should to do our best to be mindful of biases and present the facts logically. 
```

Comment:
```suggestion
Second, providing context and discussing the *meaning* behind a point estimate can help the audience glean some insight into why the data is important. For example, when comparing two values, it can be helpful to highlight if there are statistically significant differences and explain the impact and relevance of this information. This is where we, as analysts, should to do our best to be mindful of biases and present the facts logically. 
```